ui/desktop/tests/integration/helpers/order-by-test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Helper | order-by', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
```

Comment:
Could we add a few more test cases to show the different scenarios, e.g. preserves sort order if no keys are in the list, some keys are in the list, etc

Reply:
good call, added!